### ISSUE TITLE
"fabelhaftes Deutschheft": bessere/mehr Bilder

### ISSUE THREAD

**Issue by meisl** [meisl/hafis#23]
*Achtung: Verbesserungen am *Text* in #6*

* für *jedes* Buch: hochaufgelöstes Bild vom Titel
* zu *jedem* Buch ein paar Seiten Vorschau (und zwar schöner als die jetztige)

**Comment by chdoerr** [comment]
Ich habe aber keine Kamera

**Comment by meisl** [comment]
Ich hab nen Scanner, das wird glaube ich am besten. Handycamera geht aber auch. Muss man sich bisschen Mühe geben mit Licht und so. Das kriegen wir schon hin.

**Comment by chdoerr** [comment]
Okay

**Comment by meisl** [comment]
Allgemeine Gestaltung der Seite:
* Liste mit allen Büchern, jeweils mit den Daten (inkl. Untertitel) und kleinem Bild. Beim Klick auf das kleine Bild wird es groß. Momentan wirkt es ziemlich komisch, dass Deutschheft 1 so prominent ist, und die anderen nur klein ganz unten auftauchen.
* Wir könnten auch so eine Bildergalerie wie auf Puppentheater ausprobieren
* Die Vorschauen vielleicht wie bei amazon mit "Look inside"?
* Ohne Scrollen sollte gleich mind. ein Bild zu sehen sein, **aber "wie kaufen?" sollte das AUCH!**
* Für die Sache mit dem Papier ("Filzer") kann man ruhig eins rausgreifen; dieser Abschnitt kann aber ruhig weiter unten.

**Comment by meisl** [comment]
Und: warum nicht mit Zwischenüberschriften ("Überschrift 2") arbeiten? Das hilft (auch) optisch.

**Comment by meisl** [comment]
Zusammenfassung Treffen heute: (sh. #6 bezgl. Text)
* 4-6 Vorschaubilder je Buch (ich mache mehr zum Aussuchen)
* Druckerei bekommt pdfs, wohl mit eingebetteten jpgs ->heißt: wir haben selbst die Kontrolle, es liegt an uns
   - dazu wäre es für mich sehr hilfreich wenigstens eine dieser Dateien, die an die Druckerei gingen zu bekommen; möglichst von einem der Deutschhefte. Da könnte ich sehen, was in digital zu welchem Ergebnis in real geführt hat (habe ja jetzt die Bücher in echt - Danke nochmal dafür)
* bei Band 3:
   - versuchen wir nicht unter Zeitdruck zu geraten
   - möchten wir das Layout schöner und gleichmäßiger hinkriegen. Im Detail:
     - Schrift möglichst immer gleich groß und gleich "dicht" -> im Zweifel Text anpassen
     - graphische Spielereien wie kleine "Suchbilder" als Fußleiste sind toll - wenn aber überall!
     - Gute Wiedergabe des Bleistifts **ist** möglich - überall!

Wenn ich was vergessen habe, oder Ihr mit etwas nicht einverstanden seid, bitte einfach hier schreiben.

**Comment by chdoerr** [comment]
Ein (vorläufiges) Titelbild für "Elfchen" gibt es schon - das kriegst Du mit den Originalen

**Comment by meisl** [comment]
Sehr schön, bin schon gespannt :)

**Comment by meisl** [comment]
Hier das Vorschaucover von Elfchen, in zwei Versionen. In der zweiten Version hab ich Farben etwas kräftiger gemacht, damit der Unterschied zu den anderen nicht so auffällt.

Außerdem muss ich die anderen nochmal machen, damit sie
1. dasselbe (richtige) Format und Seitenverhältnis haben
2. nicht den komischen breiten weißen Rand wie jetzt, sondern diese feine schwarze Linie wie Elfchen haben

Zum Vergleich ist auch nochmal das Tierstaben-Cover dabei.

![Elfchen Vorschaucover a](https://github.com/meisl/hafis/raw/master/MyWebsite/Bilder/Deutschheft/Elfchen_Vorschaucover0_(aussen-vorn).jpg)

![Tierstaben Vorschaucover](https://github.com/meisl/hafis/raw/master/MyWebsite/Bilder/Deutschheft/Tierstaben_Aussencover-vorn.jpg)

![Elfchen Vorschaucover b](https://github.com/meisl/hafis/raw/master/MyWebsite/Bilder/Deutschheft/Elfchen_Vorschaucover0b_(aussen-vorn).jpg)

**Comment by chdoerr** [comment]
Kräftigere Farben sind besser und passen mehr zu den anderen Heften, vielleicht noch etwas kräftiger...

Kruno fand den weißen Rand sehr originell...

**Comment by meisl** [comment]
Ja, das mit den Farben fällt auf. Allerdings ist das nicht so einfach. Liegt am Original. Genau erkläre ich das morgen, mit Malprogramm (live-Vorführung).
Nur soviel:
* je intensiver man die Farben macht, desto schlechter wird der Kontrast zur Schrift. Die Buchstaben "ee oo aa", "sch" etc. in den Kreisen "saufen ab". Ich hab schon den Titel "Mein fabelhaftes..." und "Petra Cordua" entspr. nachbearbeitet, sonst wären die auch "abgesoffen"
* manche Farben werden *zu* intensiv: siehe das Gesicht und die Hände

Zum Rand: ich meine den direkt in den Bildern. Schau sie in IrfanView an, dann weisst Du was ich meine. Hier und auf der Website bemerkt man ihn als solchen erstmal nicht. Kann nicht erkennen was daran originell sein soll. Für mich ist das schlampig, weiter nix. Führt aber dazu, dass die eigtl. Bildinhalte unterschiedlich gross sind, und das sieht man durchaus auf der Website.

**Comment by chdoerr** [comment]
Aha.

Stimmt.
